app/lib/get-all-linked.js

```diff
@@ -0,0 +1,27 @@
+function getAllLinked(resource, link, query) {
+  if (!resource || !link) {
+    return;
+  }
```

Comment:
`getAllLinked` returns a promise, so maybe return `Promise.reject(new Error(‘getAllLinked: resource and link not specified’)`.